### ISSUE TITLE
WON'T FIX: Navigation Buttons are not announced by an NVDA Scren Reader

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#7400]
User Issue: T15634 - Complete button - accessibility issue
https://surveyjs.answerdesk.io/internal/ticket/details/T15634
*** 
Altought survey navigation buttons (Next/Previous/Complete) are properly read when focusing it via the TAB button, these buttons are not read when you hover it my a mouse pointer. To reproduce the issue, open a [demo](https://surveyjs.io/form-library/examples/implement-conditional-logic-to-change-question-visibility/reactjs), run the NVDA speech viewer, and hover over different buttons. For instance, Open in Theme Editor button works OK, however, hovering over the Complete button doesn't work.

**Comment by dmitrykurmanov** [comment]
It is the NVDA issue https://stackoverflow.com/questions/75064724/accessibility-aria-label-not-being-read-on-hover So we can't fix it.
